Doc/whatsnew/3.14.rst

```diff
@@ -2820,66 +2820,25 @@ CPython bytecode changes
   (Contributed by Irit Katriel in :gh:`100239`.)
 
 
-C API changes
+C API Changes
```

Comment:
Use sentence case for headers:

https://devguide.python.org/documentation/style-guide/#capitalization
```suggestion
C API changes
```